R/dataset.R

```diff
@@ -313,13 +308,7 @@ Dataset.get_by_full_path <- function(full_path) {
 #'
 #' @export
 Dataset.get_or_create_by_full_path <- function(full_path, ...) {
-    dataset = NULL
-    tryCatch({
-        dataset = do.call(Dataset.get_by_full_path, list(full_path))
-        if (!is.null(dataset)) {
-            return(dataset)
-        }
-    }, error = function(e) {})
+    dataset = do.call(Dataset.get_by_full_path, list(full_path))
```

Comment:
In this case, I think we do want the tryCatch, since this function is "get_or_create", so if create fails (now with a stop), it should continue running.

Reply:
rolled back changes to that section